components/Modal/CreateImageUpload.js

```diff
@@ -442,88 +496,297 @@ class CreateImageUploadModal extends React.Component {
+            </div>
+            <div className="pf-c-form__group">
+              <div className="pf-c-form__label pf-m-no-padding-top pf-l-flex pf-u-display-flex pf-m-justify-content-flex-start pf-m-nowrap">
+                <label htmlFor="bucket-input" className="pf-l-flex__item">
+                  <span className="pf-c-form__label-text">Amazon S3 bucket</span>
+                  <span className="pf-c-form__label-required" aria-hidden="true">
+                    &#42;
+                  </span>
+                </label>
+                <Popover
+                  id="bucket-popover"
+                  bodyContent={
+                    <React.Fragment>
+                      <FormattedMessage
+                        defaultMessage="
+                          Provide the S3 {bucket} name to which the image file will be uploaded before being imported into EC2. 
+                          The {bucket} must already exist in the Region where you want to import your image. You can find a list of {buckets} on the
+                          {bucketsPage} page in the {amazon} S3 storage service in the AWS console.
+                        "
+                        values={{
+                          bucket: "bucket",
+                          buckets: "buckets",
+                          bucketsPage: <strong>S3 buckets</strong>,
+                          amazon: "Amazon"
+                        }}
+                      />
+                    </React.Fragment>
+                  }
+                  aria-label="bucket help"
+                >
+                  <Button variant="plain" aria-label="bucket help">
+                    <OutlinedQuestionCircleIcon id="popover-icon" />
+                  </Button>
+                </Popover>
+              </div>
+              <TextInput
+                className="pf-c-form-control"
+                value={this.state.uploadSettings["bucket"]}
+                type="text"
+                id="bucket-input"
+                name="bucket"
+                onChange={this.setUploadSettings}
+              />
+            </div>
+            <div className="pf-c-form__group">
+              <div className="pf-c-form__label pf-m-no-padding-top pf-l-flex pf-u-display-flex pf-m-justify-content-flex-start pf-m-nowrap">
+                <label htmlFor="region-input" className="pf-l-flex__item">
+                  <span className="pf-c-form__label-text">
+                    <FormattedMessage defaultMessage="AWS region" />
+                  </span>
+                  <span className="pf-c-form__label-required" aria-hidden="true">
+                    &#42;
+                  </span>
+                </label>
+                <Popover
+                  id="region-popover"
+                  bodyContent={
+                    <FormattedMessage
+                      defaultMessage="Provide the AWS Region where you want to import your image. This must be the same region where the {bucket} exists."
```

Comment:
In the original upload PR, you provided this link to AWS documentation that lists the regions: https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.RegionsAndAvailabilityZones.html

We also discussed making this a select menu, which is out of scope of this PR. I'm also not sure how often region names are updated, which creates the risk that the region the user wants is not included as a select menu option. And we could always have an option to manually enter a region in that case. Which makes the control even larger in scope.

For now, I'm thinking we should at least provide the link above, and clarify that we want the "region" and not the "region name". For this latter point, we could use placeholder text "E.g. eu-central-1". And for the link, maybe this additional line in the popover would work:
> You can find a list of regions in the [AWS User Guide](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.RegionsAndAvailabilityZones.html).